neard/src/cli.rs

```diff
@@ -493,12 +493,29 @@ pub(super) struct RecompressStorageSubCommand {
     /// Directory where to save new storage.
     #[clap(long)]
     output_dir: PathBuf,
+
+    /// Keep data in ColPartialChunks column.  This is always true if node is
+    /// not an archival node.
+    #[clap(long)]
+    keep_partial_chunks: bool,
+
+    /// Keep data in ColTrieChanges column.  This is always true if node is
+    /// not an archival node.
+    #[clap(long)]
+    keep_trie_changes: bool,
```

Comment:
maybe we should put 'archival' somewhere in the name?

Reply:
Do you have idea for the names? I think the current one is descriptive enough.